presto-main/src/main/resources/webapp/tableau/presto-client.js

```diff
@@ -28,8 +28,8 @@ function StatementClient(connectionData, headerCallback, dataCallback, errorCall
     this.headers = {
         "X-Presto-User": this.user ? this.user : 'N/A',
         "X-Presto-Source": this.source,
-        "X-Presto-Catalog": this.catalog,
-        "X-Presto-Schema": this.schema,
+        "X-Presto-Catalog": this.catalog ? this.catalog : 'N/A',
```

Comment:
`N/A` is not a proper value to pass via the X-Presto-Catalog header. If anything, these headers should be omitted when the catalog and schema are unavailable.